test/suggestion.cpp

```diff
@@ -63,6 +63,10 @@ namespace {
         creator.finishZimCreation();
         return zim::Archive(this->path());
       }
+
+      std::string getPath() {
```

Comment:
Make `const`

A general advice - anything should be `const` by default. That helps to eliminate a lot of errors due to unintended change of state. Mutation must be allowed if it cannot be avoided.